Beth editorial of home & installation (#128)
neuralmagic/docs: src/content/index.mdx

> Neural Magic's Deep Sparse Platform provides models and tools needed to create sparse models and a CPU-based inference engine that runs sparse models at GPU speeds.

Reviewer: ```suggestion
The Neural Magic Platform provides models and tools needed to create sparse models and a CPU-based inference engine that runs sparse models at GPU speeds.
```

Reviewer: We are changing the name from Deep Sparse Platform to Neural Magic Platform FYI.

Using the Deep Sparse Platform, you are free to deploy your neural networks from edge to cloud&mdash;anywhere you have a CPU.

Three products accomplish these goals:

> - [DeepSparse Engine](/products/deepsparse) offers best-in-class CPU performance for dense and sparsified models.

Reviewer: ```suggestion
- [DeepSparse](/products/deepsparse) offers best-in-class CPU performance for dense and sparsified models.
```

Specifically for sparse models, it delivers better than GPU performance in many use cases.
The performance is achieved by leveraging technology around the unique cache hierarchy of CPUs for faster memory access and sparsification techniques to reduce the number of FLOPs.

>   The DeepSparse Engine is available in two editions:

Reviewer: ```suggestion
 DeepSparse is available in two editions:
```

>   - The [Community Edition](/products/deepsparse) is open-source and free for evaluation, research, and non-production use with our [Engine Community License](https://neuralmagic.com/legal/engine-license-agreement/).

Reviewer: ```suggestion
  - [DeepSparse Community](/products/deepsparse) is open-source and free for evaluation, research, and non-production use with our [DeepSparse Community License](https://neuralmagic.com/legal/engine-license-agreement/).
```

>   - The [Enterprise Edition](/products/deepsparse-ent) requires a Trial License or [can be fully licensed](https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications.

Reviewer: ```suggestion
  - [DeepSparse Enterprise](/products/deepsparse-ent) requires a Trial License or [can be fully licensed](https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications.
```
